lib/bridge/strain.rb

```diff
@@ -11,13 +11,13 @@ def to_s
     end
 
 
-    Club    = new(0, :club     ).freeze
```

Comment:
Oopsie. I really don't mean to be going renegade on formatting. (It's a combination of over reliance on IDE auto-formatting and unmerited respect for its "warning"s regarding formatting.) It's part of what makes me feel like I'm going about this all wrong and I should just trash this PR, start two fresh repos for separate gems bridge-pbn-rb and bridge-stats-rb which depend on the domain objects here. What do you think?